api/src/server.ts

```diff
@@ -37,6 +37,7 @@ class ApiServer {
     const connectionParams = extractConnectionParams(this._config);
     const dbConnection = new Connection(connectionParams);
     this._dbConnection = dbConnection;
+    console.log(`Using database at ${connectionParams.host}`);
```

Comment:
Would you like this console.log statement to stay in the code?

Reply:
I figured it might be useful for the app to report which database its connecting to, given we've recently been changing that around.